build.sc

```diff
@@ -52,6 +52,8 @@ object Settings {
 object Deps {
 
   // The Scala version to use
+  // When updating, run "Publish Bridges" Github Actions for the new version
+  // and then add to it `bridgeScalaVersions`
```

Comment:
I think we should auto-add all required verions (`scalaVersion`, `workerScalaVersion212`, probably `scalaVersionForScoverageWorker1`) to `bridgeScalaVersions`.